invenio_files_rest/models.py

```diff
@@ -1306,6 +1306,25 @@ def delete(cls, bucket, key):
             return cls.create(bucket, key)
         return None
 
+    @classmethod
+    def remove_many(cls, version_ids, bucket):
+        """Permanently removes specific object versions from the database.
+
+        :param version_ids: list of version_id.
+        :param bucket: The bucket (instance or id) to delete the object from.
+        """
+        bucket = as_bucket(bucket)
+        if bucket.locked:
+            raise BucketLockedError()
+        object_versions = cls.query.filter(
+            cls.version_id.in_(version_ids), cls.bucket_id == bucket.id
+        )
+        for object_version in object_versions.all():
+            bucket.size -= object_version.file.size
+        object_versions.delete()
```

Comment:
judging from the delete function above, shouldn't we do the same? ie. create an empty object version instead of deleting ? I think this might be hard delete, removing all the revisions (I might be wrong)